recipes/snakemake/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - docutils
     - pyyaml
     - requests >=2.8.1
-    - dropbox >=5.2
```

Comment:
Good point. I also forgot to do that in the other PR. I can add those changes here  or in a separate PR if you like